tests/phpunit/tests/post/wpTrashPost.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+/**
+ * @group post
+ * @covers ::wp_trash_post
+ */
+class Tests_Post_wpTrashPost extends WP_UnitTestCase {
```

Comment:
```suggestion
class Tests_Post_WpTrashPost extends WP_UnitTestCase {
```
`wp` should be `Wp` here. [See the handbook entry on Test Classes](https://make.wordpress.org/core/handbook/testing/automated-testing/writing-phpunit-tests/#test-classes) and [this ticket about 'wp' vs 'Wp' for test class names](https://core.trac.wordpress.org/ticket/56846).